packages/codecs-core/src/__tests__/reverse-codec-test.ts

```diff
@@ -10,23 +10,29 @@ describe('reverseCodec', () => {
         const s = (size: number) => reverseCodec(fixCodecSize(base16, size));
 
         // Encode.
-        expect(s(1).encode('00')).toStrictEqual(b('00'));
-        expect(s(2).encode('00ff')).toStrictEqual(b('ff00'));
-        expect(s(2).encode('ff00')).toStrictEqual(b('00ff'));
-        expect(s(4).encode('00000001')).toStrictEqual(b('01000000'));
-        expect(s(4).encode('01000000')).toStrictEqual(b('00000001'));
-        expect(s(8).encode('0000000000000001')).toStrictEqual(b('0100000000000000'));
-        expect(s(8).encode('0100000000000000')).toStrictEqual(b('0000000000000001'));
-        expect(s(32).encode(`01${'00'.repeat(31)}`)).toStrictEqual(b(`${'00'.repeat(31)}01`));
-        expect(s(32).encode(`${'00'.repeat(31)}01`)).toStrictEqual(b(`01${'00'.repeat(31)}`));
+        expect(s(1).encode('99')).toStrictEqual(b('99'));
```

Comment:
I replaced `00` in here with `99`

* visually distinct from `ff`
* catches bugs – forgetting to write in the middle value would still pass the test if the value in the middle happened to be `00`